db/migrate/20200103012916_create_land_schema.rb

```diff
@@ -0,0 +1,225 @@
+class CreateLandSchema < ActiveRecord::Migration[5.0]
+  QUERY_PARAMS = %w[
+    app
+    ad_type
+    ad_group
+    bid_match_type
+    campaign
+    content
+    creative
+    device_type
+    experiment
+    keyword
+    match_type
+    medium
+    network
+    placement
+    position
+    search_term
+    source
+    subsource
+    target
+  ]
+
+  def schema
+    Land.config.schema
+  end
+
+  def up
+    execute "CREATE SCHEMA #{schema};"
+
+    with_options schema: schema do |t|
+      # Query params
+      t.create_lookup_tables(*QUERY_PARAMS.map(&:pluralize))
+
+      # User agent
+      t.create_lookup_table  :devices
+      t.create_lookup_tables :user_agent_types, :browsers, :platforms, small: true
+
+      # HTTP
+      t.create_lookup_tables :domains, :paths, :query_strings
+      t.create_lookup_tables :http_methods, :mime_types, small: true
+
+      t.create_lookup_table :event_types, small: true
+    end
+
+    execute %[CREATE EXTENSION "uuid-ossp";]
```

Comment:
Let's make sure we can use this with Elastic Beanstalk + RDS without too much fuss